src/Tizen.NUI/src/public/ParticleSystem/ParticleEmitter.cs

```diff
@@ -0,0 +1,572 @@
+        /// </summary>
+        /// <remarks>
+        /// Initial number of particles to be emitted immediately after emitter starts. It allows
+        /// initial burst emission. By default it's set to 0.
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint InitialParticleCount
+        {
+            get
+            {
+                var value = Interop.ParticleEmitter.GetInitialParticleCount(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                return value;
+            }
+            set 
+            {
+                Interop.ParticleEmitter.SetInitialParticleCount(SwigCPtr, value);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve(); 
+            }
+        }
+        
+        /// <summary>
+        /// Limit of active particles in the system
+        /// </summary>
+        /// <remarks>
+        /// Active particles in the system can be limited without changing <see cref="ParticleCount"/>.
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public uint ActiveParticleLimit
+        {
+            get{
+                var value = Interop.ParticleEmitter.GetActiveParticlesLimit(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                return value;
+            }
+            set 
+            {
+                Interop.ParticleEmitter.SetActiveParticlesLimit(SwigCPtr, value);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+            
+        }
+
+        /// <summary>
+        /// Gets/sets blending mode for particle renderer
+        /// </summary>
+        /// <remarks>
+        /// Currently two blending modes are supported: Additive and Screen (advanced blending mode).
+        /// <see cref="ParticleBlendingMode"/>
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ParticleBlendingMode RendererBlendingMode
+        {
+            get
+            {                
+                var value = Interop.ParticleEmitter.GetBlendingMode(SwigCPtr);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                return (ParticleBlendingMode)value;
+            }
+            set
+            {
+                Interop.ParticleEmitter.SetBlendingMode(SwigCPtr, value);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            }
+        }
+        
+        /// <summary>
+        /// Sets texture to be used by the renderer
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Texture RendererTexture
```

Comment:
```suggestion
        public strimg TextureResourceUrl
```

This is soke kind of suggestion. How about make this property type as `string`, and implement what demo app doing, internally.
Standard UI developer dont care what is 'Texture' and how we can load it.

Reply:
Maybe it is not an neccesary job. You can ignore it

Reply:
Oh I already did it